server/plugin.go

```diff
@@ -158,17 +187,16 @@ func (p *Plugin) createPost(userID, channelID, message string) {
 		p.API.LogError("Error fetching Jenkins user details", "err", userInfoErr.Error())
 		return
 	}
+
 	slackAttachment := generateSlackAttachment(message)
 	slackAttachment.Pretext = fmt.Sprintf("Initiated by Jenkins user: %s", userInfo.Username)
 	post := &model.Post{
-		UserId:    userID,
+		UserId:    p.botUserID,
 		ChannelId: channelID,
 		Type:      model.POST_DEFAULT,
 		Props: map[string]interface{}{
-			"from_webhook":      "true",
-			"override_username": jenkinsUsername,
-			"override_icon_url": p.getConfiguration().ProfileImageURL,
-			"attachments":       []*model.SlackAttachment{slackAttachment},
+			"from_webhook": "true",
```

Comment:
Same question

Reply:
Are you talking about line 190? I don't see a blank line here

Reply:
Missed one comment in my review, sorry. See https://github.com/mattermost/mattermost-plugin-jenkins/pull/15#discussion_r305611578

Reply:
No worries :slightly_smiling_face:

Thanks for pointing this out @hanzei. I've removed the prop from both places.